jujugui/static/gui/src/app/store/env/api.js

```diff
@@ -2861,6 +2861,9 @@ YUI.add('juju-env-api', function(Y) {
           var response = data.response;
           transformedData.envs = response['user-models'].map(function(value) {
             var model = value.model;
+            if (!model) {
```

Comment:
Tracked this down to bad data being returned by `sandbox.js`, so I removed this logic and made the fix there.